modules/libcom/src/misc/epicsTypes.h

```diff
@@ -41,7 +41,7 @@ typedef enum {
  * These are sufficient for all our current archs
  * @{
  */
-typedef char            epicsInt8;
+typedef signed char     epicsInt8;
```

Comment:
I think it will maximize consistency. Compare with stdint.h:
typedef signed char             int8_t;

Also, you pointed out earlier that for C++ `char`, `signed char` and `unsigned char` are three different things, not two. Conditional definition will increase the number of cross platform inconsistencies.

Reply:
I say this as a practical consideration.  It might help with concerns by @anjohnson about breaking downstream code on targets where `char` is and always has been, signed.

Personally, I wish that 5e0bf53fecb332a53656ca5e68a62823a6893d59 had gone the other way, and defined `epicsInt*` in terms of `int*_t`, but I understand what the motivations where (little usage of C99).

Reply:
I hope it's unlikely that someone would use `epicsInt8` in their code as a synonym for `char` and pass such an array to OS routines like `strlen()` and `strcpy()` which take a `char *`. If they did in C code this change should only result in build warnings on arch's where `char` is signed, but C++ code could fail to compile — however I guess that may be what we would want to happen in that case?

Reply:
With this change, the situation you describe would certainly become a compilation error.  I don't see any way to avoid a breakage in this situation though.  The fix (an explicit cast) is simple and backwards compatible.

The potentially nastier sort of bug w/ c++ is when the type of a virtual method or template changes as these might not result in a compilation error.  I've been bitten by this when porting code for a library which switched a base class method from `long` -> `size_t` so that my overloaded method no longer matched.  Still, this possibility isn't enough to deter me.